packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -417,7 +439,8 @@ function ActiveRequests({ votingAccount, votingGateway, snapshotContract }) {
           [encryptionTimestamp]: {
             salt,
             price,
-            identifierPrecision
+            identifierPrecision,
+            ancillaryData
```

Comment:
Nice adding this here, not sure how you would have thought to do so without remembering about this button but +1

Reply:
@nicholaspai actually your run_tests script is very thorough and once i added requests with both ancillary and non ancillary data I had to do a lot more updates. pretty much every change in this pr was found thanks to that script 👍